ivy/functional/frontends/numpy/logic/array_contents.py

```diff
@@ -36,3 +38,24 @@ def allclose(a, b, rtol=1e-05, atol=1e-08, equal_nan=False):
 def isclose(a, b, rtol=1e-05, atol=1e-08, equal_nan=False):
     a, b = promote_types_of_numpy_inputs(a, b)
     return ivy.isclose(a, b, rtol=rtol, atol=atol, equal_nan=equal_nan)
+
+
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+@handle_numpy_casting_special
+def isreal(
```

Comment:
not sure if I missed anything, but I believe `isreal` only have one arguments? 

the link I referenced is: https://numpy.org/doc/stable/reference/generated/numpy.isreal.html